client/www/app/docs/next-ssr/page.md

```
@@ -110,7 +110,7 @@ Note that we also included `firstPartyPath`. This lets us sync auth between clie
 
 ### 2. Sync auth
 
-Syncing auth is covered in more detail [here](/docs/backend#syncing-auth), but we'll reproduce the main steps to this tutorial easy to follow.
+Syncing auth is covered in more detail [here](/docs/backend#syncing-auth), but we'll reproduce the main steps to make this tutorial easy to follow.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)